<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Director's Notes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVideoWidget" name="wgt_player" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1081</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play_pause">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play / Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_duration">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>510</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-- / --</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_note">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_comment">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>880</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>880</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>880</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_tag">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentText">
     <string>All</string>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>570</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Tags</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QTreeWidget" name="wgt_notes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>851</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="tagsList">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>590</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <zorder>wgt_player</zorder>
   <zorder>btn_play_pause</zorder>
   <zorder>lab_duration</zorder>
   <zorder>btn_note</zorder>
   <zorder>btn_comment</zorder>
   <zorder>btn_edit</zorder>
   <zorder>label</zorder>
   <zorder>wgt_notes</zorder>
   <zorder>tagsList</zorder>
   <zorder>combo_tag</zorder>
   <zorder>btn_del</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSession">
     <property name="title">
      <string>Start Session</string>
     </property>
     <addaction name="separator"/>
     <addaction name="new_sess"/>
     <addaction name="load_sess"/>
    </widget>
    <addaction name="menuSession"/>
    <addaction name="btn_export"/>
    <addaction name="btn_save_as"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionTags"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuManage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="btn_select">
   <property name="text">
    <string>Load Video</string>
   </property>
  </action>
  <action name="btn_data">
   <property name="text">
    <string>Load session data</string>
   </property>
  </action>
  <action name="btn_new">
   <property name="text">
    <string>New Session</string>
   </property>
  </action>
  <action name="new_sess">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="load_sess">
   <property name="text">
    <string>Load from file</string>
   </property>
  </action>
  <action name="btn_export">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="btn_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="btn_save_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionTags">
   <property name="text">
    <string>Tags</string>
   </property>
  </action>
  <action name="clear_all">
   <property name="text">
    <string>Clear All Notes</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
